src/i18n/locales/de-DE.json

```diff
@@ -0,0 +1,167 @@
+{
+  "global.api.unhealthy": "Verbindung mit dem Franz Online Service fehlgeschlagen.",
+  "global.notConnectedToTheInternet": "Du bist nicht mit dem Internet verbunden.",
+  "welcome.signupButton": "Kostenloses Konto erstellen",
+  "welcome.loginButton": "Anmelden",
+  "welcome.slogan": "Kommunikation, die für Dich funktioniert",
+  "login.headline": "Anmelden",
+  "login.email.label": "E-Mail-Adresse",
+  "login.password.label": "Passwort",
+  "login.submit.label": "Anmelden",
+  "login.invalidCredentials": "E-Mail-Adresse oder Passwort ungültig",
+  "login.tokenExpired": "Deine Sitzung ist abgelaufen, bitte melde dich erneut an.",
+  "login.serverLogout": "Deine Sitzung ist abgelaufen, bitte melde dich erneut an.",
+  "login.link.signup": "Kostenloses Konto erstellen",
+  "login.link.password": "Passwort zurücksetzen",
+  "password.headline": "Passwort zurücksetzen",
+  "password.email.label": "E-Mail-Adresse",
+  "password.submit.label": "Absenden",
+  "password.noUser": "Es wurde kein Benutzer mit dieser E-Mail-Adresse gefunden",
+  "password.successInfo": "Bitte überprüfe Deine E-Mail-Adresse",
+  "password.link.signup": "Kostenloses Konto erstellen",
+  "password.link.login": "Anmelden",
+  "signup.headline": "Registrierung",
+  "signup.firstname.label": "Vorname",
+  "signup.lastname.label": "Nachname",
+  "signup.email.label": "E-Mail-Adresse",
+  "signup.company.label": "Unternehmen",
+  "signup.password.label": "Passwort",
+  "signup.submit.label": "Konto erstellen",
+  "signup.link.login": "Du hast bereits ein Konto? Melde Dich an.",
+  "signup.emailDuplicate": "Ein Benutzer mit dieser E-Mail-Adresse existiert bereits.",
+  "signup.legal.info": "Mit der Erstellung eines Franz Kontos, akzeptierst Du die",
+  "signup.legal.terms": "Nutzungsbedingungen",
+  "signup.legal.privacy": "Datenschutzerklärung",
+  "pricing.headline": "Unterstütze Franz",
+  "pricing.support.label": "Wie möchtest Du Franz unterstützen?",
```

Comment:
du?

Reply:
I chose the capitalized version on pursed, since it is considered the slightly more formal version of "du", "dich", etc.

Reply:
I am happy to change the PR, if you considered the informal version more appropriate for Franz

Reply:
I didn't know a capitalized version of du is allowed in German. Never to old to learn ;)
But perhaps someone who is German can better approve your code. I won't take a side in this. 👍

Reply:
I think the informal version is more appropriate. The Duden says you should use the capitalized version only in letters, mails or the like.  [Source](https://www.duden.de/sprachwissen/sprachratgeber/gross-oder-kleinschreibung-von--em-du-du--em--und--em-ihr-ihr--em--1)